src/utils/breach-resolution.js

```diff
@@ -124,17 +137,24 @@ function appendBreachResolutionChecklist (userBreachData) {
  * @param {object} args contains necessary variables for the fluent file
  *  - companyName
  *  - breachedCompanyUrl
+ * @param {Partial<{ countryCode: string, hideBreachLink: boolean }>} options
  * @returns {Map} map of relevant breach resolution recommendations
  */
-function getResolutionRecsPerBreach (dataTypes, args) {
+function getResolutionRecsPerBreach (dataTypes, args, options = {}) {
   const filteredBreachRecs = {}
 
   // if datatypes is empty or null, return general only.
   if (!dataTypes?.length) dataTypes = [BreachDataTypes.General]
 
   // filter breachResolutionDataTypes based on relevant data types passed in
   for (const [key, value] of Object.entries(breachResolutionDataTypes)) {
-    if (dataTypes.includes(key)) {
+    if (
+      dataTypes.includes(key) &&
+      // Hide the security question or password resolution if we can't link to the breached site:
+      (![BreachDataTypes.Passwords, BreachDataTypes.SecurityQuestions].includes(key) || !options.hideBreachLink) &&
+      // Hide resolutions that apply to other countries than the user's:
+      (!options.countryCode || !Array.isArray(value.applicableCountryCodes) || value.applicableCountryCodes.includes(options.countryCode.toLowerCase()))
+    ) {
```

Comment:
PM related question: Rather than hide the entire recommendation for specific links that may not be applicable, would there be less complexity and more content if the copy was simply changed to account for cases where a link doesn't apply?

For example, simply appending "or similar" to the following:
> Protect your email with an email masking service like { $firefoxRelayLink }

I suppose "like" above already suggests a user might use an alternative.  My point is the user might consider an email masking service regardless if it's Relay or other – it's still a good recommendation?

Additionally, I recall some logic @mansaj might have added that detects if no specific recommendation is available and then defaults to a general recommendation.  I haven't seen or tested if this PR accounts for that (might be non-issue), but my concern would be of that general default logic not firing after we remove a recommendation due to locale, and then the user is left with zero recommendations.